README.Rmd

```diff
@@ -16,14 +16,16 @@ knitr::opts_chunk$set(
 )
 ```
 
-# substrait
+# Substrait R Package for Generating Cross-language Specification of Data Compute Operations
 
 <!-- badges: start -->
 [![R-CMD-check](https://github.com/voltrondata/substrait-r/workflows/R-CMD-check/badge.svg)](https://github.com/voltrondata/substrait-r/actions)
 [![Codecov test coverage](https://codecov.io/gh/voltrondata/substrait-r/branch/master/graph/badge.svg)](https://app.codecov.io/gh/voltrondata/substrait-r?branch=master)
 <!-- badges: end -->
 
-The goal of substrait is to provide an R interface to the [Substrait](https://substrait.io) cross-language serialization for relational algebra. This is an experimental package that is under heavy development!
+The goal of the Substrait R package is to provide an R interface to  [Substrait](https://substrait.io), a cross-language specification for data compute operations. The Substrait R package allows you to construct a Substrait plan from R for evaluation by a Substrait consumer, such as Arrow or DuckDB.
```

Comment:
Thanks for this update, I think the language there helps disambiguate the two projects.

I was thinking about how other R packages deal with the "identically-named R package and thing it exposes" problem, and I had a look at [the DESCRIPTION file for DuckDB, who lead with what DuckDB is and then talk about what the package does](https://cran.r-project.org/web/packages/duckdb/index.html):

> The DuckDB project is an embedded analytical data management system with support for the Structured Query Language (SQL). This package includes all of DuckDB and a R Database Interface (DBI) connector.

Actually, since then, I looked at [the Arrow R package](https://github.com/apache/arrow/tree/master/r), and we do the same there:

> [Apache Arrow](https://arrow.apache.org/) is a cross-language development platform for in-memory and larger-than-memory data. It specifies a standardized language-independent columnar memory format for flat and hierarchical data, organized for efficient analytic operations on modern hardware. It also provides computational libraries and zero-copy streaming, messaging, and interprocess communication.

> The arrow R package exposes an interface to the Arrow C++ library, enabling access to many of its features in R. It provides low-level access to the Arrow C++ library API and higher-level access through a {dplyr} backend and familiar R functions.

What do you think to re-wording to something a bit more like this? I feel like "the Substrait R package" is clunky but necessary (in the same way that the Arrow docs use "the Arrow R package" a lot too), but that leading with a description of what Substrait itself is might be more descriptive?

Reply:
Great call to look at what is already being done in this situation. I will edit to mimic this approach.